docs/INSTALL.md

```diff
@@ -28,7 +28,7 @@ your dotfiles with the single command:
 Or on systems with Powershell, you can use this command:
 
     # To install in ./bin
-    (iwr https://git.io/chezmoi.ps1).Content | powershell -c -
+    (iwr -UseBasicParsing https://git.io/chezmoi.ps1).Content | powershell -c -
```

Comment:
I would suggest adding -NoLogo and -NoProfile to the powershell command at the end to avoid loading unnecessary output and loading of the profile just for the installation. Much faster if you have a bit of stuff in the profile.